DEVELOPMENT.md

```diff
@@ -78,6 +78,9 @@ utils/pull-nixpkgs.sh <NIXPKGS-DIR>
 ```
 
 * Visually review changes created by `pull-nixpkgs.sh` script
+```
+git diff
+```
```

Comment:
A bit late, but if you want a piece of code (or block quote, or other) to be *part of* the list item above it (especially useful in ordered lists), you need to add an empty line between them and indent the content by three spaces. Example:

1. My code:

   ```
   some code
   ```
1. Note how this gets numbered 2, because it's now treated as part of the list above.

Markdown:

<pre>
1. My code:

   ```
   some code
   ```
1. Note how this gets numbered 2, because it's now treated as part of the list above.
</pre>

Reply:
Thank you for suggestion. I am aware that I don't follow md syntax correctly. I'll try to improve it.